src/DI/ServiceDefinition.php

```diff
@@ -308,33 +308,17 @@ public function getImplementMode()
 
 
 	/** @deprecated */
-	public function setImplementType($type)
-	{
-		trigger_error(__METHOD__ . '() is deprecated, use setImplementMode()', E_USER_DEPRECATED);
-		return $this->setImplementMode($type);
-	}
-
-
-	/** @deprecated */
-	public function getImplementType()
-	{
-		trigger_error(__METHOD__ . '() is deprecated, use getImplementMode()', E_USER_DEPRECATED);
-		return $this->implementMode;
-	}
-
-
-	/** @return static */
 	public function setInject($state = TRUE)
 	{
-		//trigger_error(__METHOD__ . '() is deprecated.', E_USER_DEPRECATED);
+		trigger_error(__METHOD__ . "() is deprecated, use addTag('inject')", E_USER_DEPRECATED);
```

Comment:
@dg I think it's because some people used to enable inject everywhere. (very bad practice)